### PR TITLE
Touts Components (Breadcrumbs WIP) 

### PR DESCRIPTION
**Pull request by MaximumCrash**
### Attention (CC)
@twblack88 
@andytudhope 
@isaacpearl 

## Summary of changes
- Minor fixes and tweaks in places that needed.
- Added logic to ensure ALL external links will be HTTPS. 
- Fixes to status banner z indexing causing conflicts with images. 
- Fixed InfoBlock content being improperly spaced. 
- Updated poolparty script to remove auto labeling. 

## Motivation & Context
- Issues #51 

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation Update
- [ ] Other (please describe):

## Does this introduce a breaking change?

_Breaking change is defined as_ **"A fix or feature that would cause existing functionality to change."**

- [x] Yes
- [ ] No

I've found a bug with the Breadcrumbs component where if you're on a page "like bounties" and say the page prior to it (in Breadcrumbs logic) doesn't exist in your current locale it will crash. This is because Breadcrumbs consider your site structure to be near perfect, but doesn't take into consideration deeper pages that may be missing in between. Because the sidebar already does this, I just need to apply a similar fallback logic to the breadcrumbs component. 

This is an edge case that exists on R2D already.

TLDR: If english has pages /en/learn/dai and french has /en/[DNE]/dai. And the learn page doesn't exist on your current (french) locale. The bread crumbs crash because it's trying to render a link to data that doesn't exist. 

## Checklist:

- [x] My code follows the code style of this project.
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added/changed necessary documentation (if appropriate)
- [ ] All new and existing tests passed.

## How Has This Been Tested?
Locally

## Screenshots (if appropriate):
Screen Shot 2020-06-24 at 4 19 08 PM
---
Screen Shot 2020-06-24 at 4 19 12 PM
---
Screen Shot 2020-06-24 at 4 19 19 PM
---
Screen Shot 2020-06-24 at 4 22 39 PM
---
Screen Shot 2020-06-24 at 4 22 47 PM
---
Screen Shot 2020-06-24 at 4 22 53 PM

### Screenshots to MDX 

```md
# Testing the Tout Component

## This is an Tout component tester. Nothing to see here.


<Tout>

# Hiya

</Tout>

<Tout alt>

Hello World

</Tout>

<Tout color="red">

Hello World

</Tout>


<Tout stroke>

# Hello World ?

</Tout>

<Tout alt stroke>

# Hello World ?

</Tout>

<Tout alt stroke color="red">

# Hello World ?

</Tout>

<Tout fade>

# Hello World ?

</Tout>

<Tout fade alt>

# Hello World ?

</Tout>

<Tout fade alt stroke>

# Hello World ?

</Tout>

<Tout fade color="black">

# Hello World ?

</Tout>

<Tout image>

<Box>

## Working on a bounty

See what's expected of you if you work on a Maker community bounty

[Working on a bounty](http://makerdao.com)

</Box>


</Tout>


<Tout image alt>

<Box>

## Working on a bounty

See what's expected of you if you work on a Maker community bounty

[Working on a bounty](http://makerdao.com)

</Box>


<Box>

## Say hi!

The best place to start is in our chat. Say hello and get to know us.

<Link icon="rocketchat">Community chat</Link>

</Box>

<Box>

## Working on a bounty

See what's expected of you if you work on a Maker community bounty

[Working on a bounty](http://makerdao.com)

</Box>

</Tout>

<Tout image>

Hello

Goodbye 

</Tout>

<Tout image>

<Box>

## Working on a bounty

See what's expected of you if you work on a Maker community bounty

[Working on a bounty](http://makerdao.com)
</Box>

</Tout>


<Tout image>

<Box>

## Working on a bounty

See what's expected of you if you work on a Maker community bounty

[Working on a bounty](http://makerdao.com)
</Box>

How's it going.

## Pretty Good

# How about you?

</Tout>


<Tout image alt>

<Box>

## Working on a bounty

See what's expected of you if you work on a Maker community bounty

[Working on a bounty](http://makerdao.com)
</Box>

How's it going.

## Pretty Good

# How about you?

</Tout>

```

### How it works

```alt``` prop applies yellow alternative color
```color``` prop allows user to apply custom css colors/hex codes for the background. If a custom color prop is provided the copy will change to check against HexLuminance rules for whether it should be dark or light text. 
**NOTE: Based on the prop (stroke/fade) the HexLuminance check changes. This is because stroke doesn't have a background and fades do, but in both cases a color prop can be provided.**

```stroke``` prop removes the background and applies a colored border of (default, alt, or color prop)
```fade``` prop fades the background from the users specified color (or default/alt variants) to transparent. 
```image``` prop applies 1 of 2 static images depending on the existance of the alt prop. (pulled from makerdao.com to match specs) if no image prop is provided a user may also provide an image as the **last** child of the tout for custom control. (See InfoBlocks on this philosophy)

If there's only **one** child (following the child rule) then the Tout will be full length. If there are 2 or more, they will be stacked 2 per row with **no** centering of an odd element. 

This component is relatively straight forward, not much out of left field with this one.

### CODE REVIEW

**Review by andytudhope — APPROVED** [review]
Will get this documented tomorrow. Likely gonna put touts in the advanced section, because we need to cover children first, then can move into discussions of hexLuminance and all the fancy stuff which can be well-illustrated with this component.
